packages/block-editor/src/components/colors-gradients/control.js

```diff
@@ -27,19 +30,6 @@ const colorsAndGradientKeys = [
 	'disableCustomGradients',
 ];
 
-const TAB_COLOR = {
-	name: 'color',
-	title: __( 'Solid' ),
-	value: 'color',
-};
-const TAB_GRADIENT = {
-	name: 'gradient',
-	title: __( 'Gradient' ),
-	value: 'gradient',
-};
-
-const TABS_SETTINGS = [ TAB_COLOR, TAB_GRADIENT ];
```

Comment:
I know that most of this configuration is not necessary anymore with the new `Tabs` component, but I wonder if it would be beneficial to still have the tab IDs as constants, to make it more clear throughout the code that certain values need to be the same

Something like `const TAB_IDS = { color: 'color', gradient: 'gradient' }`

What do folks think?

Reply:
Some more specificity and purity wouldn't hurt 👍 Such can definitely help with readability too.